.gitignore

```diff
@@ -40,4 +40,6 @@
 dependency-reduced-pom.xml
 target/
 tests.log*
-**/nb-configuration*.xml
\ No newline at end of file
+**/nb-configuration*.xml
+/underfs/s3/nbproject/
+/underfs/oss/nbproject/
```

Comment:
if we ignore `**/nbproject/` instead, it should ignore those two nbproject directories as well as any other nbproject directories that might be needed in other projects. The `**` matches any number of directories.